frontend/src/modules/signup-old/view.jsx

```diff
@@ -68,8 +82,12 @@ const SignupView = ({ ...props }) => {
                     className="green"
                     onClick={() => {
                       setTypeSignUp("stakeholder");
-                      loginWithPopup({ screen_hint: "signup" });
-                      //                  history.push("/entity-signup");
+                      console.log(profile.about, Boolean(profile.about));
+                      if (isRegistered(profile)) {
```

Comment:
Need to make sure of this condition,
I think this if condition should be `if (!isRegistered(profile))`, then go to register?

Reply:
thanks for catching that!

Reply:
finally using isAuthenticated